docroot/modules/custom/civictheme_migrate/src/Component/ManualList.php

```diff
@@ -20,10 +20,13 @@ public static function getSrcFields(): array {
    * {@inheritdoc}
    */
   protected function prepareData($data, array $context): array {
-    if (!empty($data['children']['cards']['children'])) {
+    $data = $data['children'] ?? $data;
+
+    if (!empty($data['cards']['children'])) {
       $cards = [];
       foreach ($data['children']['cards']['children'] as $children) {
-        foreach ($children as $card_type => $card_info) {
+        foreach ($children as $card_type => $card_children) {
```

Comment:
Merlin is exporting this inside child instead of a direct object so need to process it before creating a component.